lib/search_api.rb

```diff
@@ -53,7 +54,15 @@ def is_scoped?
     end
 
     def scope_object_link
-      params.filter('manual').first
+      @link ||= params.filter('manual').first
```

Comment:
Better to have memoisation variables named the same as the method they're memoising.